src/Rings/groebner/general.jl

```diff
@@ -120,8 +120,10 @@ function standard_basis(I::MPolyIdeal; ordering::MonomialOrdering = default_orde
       K_hom = H(K)
       gb_hom = IdealGens(H.(gens(gb)))
       gb_hom.isGB = true
+      gb_hom.ord = degrevlex(S)
+      singular_assure(gb_hom)
```

Comment:
What is this `singular_assure` call doing? Can we safely remove it? I mean, I can of course just remove it and try what happens but I am worried about this having invisible consequences (i.e. instead of triggering an error it might introduce a subtle bug), simply because I find this entire code segment somewhat confusing.